doc/cli/operator-sdk_olm-catalog_gen-csv.md

```diff
@@ -21,9 +19,9 @@ operator-sdk olm-catalog gen-csv [flags]
 
 ```
       --csv-channel string     Channel the CSV should be registered under in the package manifest
-      --csv-config string      Path to CSV config file. Defaults to deploy/olm-catalog/csv-config.yaml
       --csv-version string     Semantic version of the CSV
       --default-channel        Use the channel passed to --csv-channel as the package manifests' default channel. Only valid when --csv-channel is set
+      --exclude strings        Paths to exclude from CSV generation, ex. "deploy/prod,deploy/test"
```

Comment:
I think should be e.g instead of e.x . Am I right?

Reply:
Also, could we not make it be gen with the array type ?

Reply:
We can't modify the type strings AFAIK. The example should provide enough of a clue though.